build.sbt

```diff
@@ -4274,7 +4274,7 @@ lazy val `os-environment` =
     .in(file("lib/java/os-environment"))
     .enablePlugins(JPMSPlugin)
     .settings(
-      frgaalJavaCompilerSetting,
+      // frgaalJavaCompilerSetting,
```

Comment:
We need access to [java.lang.foreign API](https://docs.oracle.com/en/java/javase/24/docs/api/java.base/java/lang/foreign/package-summary.html) which is only available since JDK 22. Frgaal doesn't have a snapshot of such API in its current version.

Reply:
- Resolved in ebdca0c
- we should try to compile with regular Javac - it may make `sbt` was faster - as it users `javac` in the same JVM...
- we only have to built upon ebdca0c and use regular Javac for (almost) all `frgaalCompilerSettings`